satsense/image.py

```diff
@@ -276,32 +272,48 @@ def save(self, filename_prefix='', extension='nc'):
                 self._save_as_netcdf(data, filename, window)
             elif extension.lower() == 'tif':
                 self._save_as_tif(data, filename, window)
+        return filenames
 
     def _save_as_netcdf(self, data, filename, window):
         """Save feature vector as NetCDF file."""
+        if len(data.shape) == 2:
+            height, width = data.shape
+            value_length = 1
+            data = data[:, :, np.newaxis]
+        elif len(data.shape) == 3:
+            height, width, value_length = data.shape
+
         with Dataset(filename, 'w') as dataset:
+            # Metadata
+            dataset.history = 'Created ' + time.ctime(time.time())
```

Comment:
It would be nice to have some of these global attributes of the netcdf file also available as tags for the tiff file (where relevant).

Reply:
Will add these